Source/HTTP/Android/android_http_request.cpp

```diff
@@ -0,0 +1,355 @@
+#include "pch.h"
+
+#include "android_http_request.h"
+
+#include <httpClient/httpClient.h>
+
+/* static */ JavaVM* HttpRequest::s_javaVM = nullptr;
```

Comment:
>/* static */ [](start = 0, length = 12)

why /* static */ everywhere?  seems like you should just remove this

Reply:
as a reminder these are static, since it's not part of the definition, only the declaration

---
In reply to: [189750654](https://github.com/Microsoft/libHttpClient/pull/165#discussion_r189750654) [](ancestors = 189750654)